content/docs/querying/operators.md

```diff
@@ -195,7 +196,7 @@ If the metric `http_requests_total` had time series that fan out by
 `application`, `instance`, and `group` labels, we could calculate the total
 number of seen HTTP requests per application and group over all instances via:
 
-    sum(http_requests_total) by (application, group)
+    sum(http_requests_total) without (instance)
```

Comment:
`by` works okay when you know a-priori what all of your labels are, and don't need to work with any other label taxonomy. `without` makes things a bit more nuanced in terms of naming (you'll need to distinguish between target and instrumentation labels), but allows much easier sharing of rules and is generally less error prone.